Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Generic/PlantEnable.mo

```diff
@@ -0,0 +1,395 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Generic;
+block PlantEnable "Sequence to enable and disable plant"
+
+  parameter Boolean haveWSE = true
+    "Flag to indicate if the plant has waterside economizer";
+  parameter Real schTab[4,2] = [0,1; 6*3600,1; 19*3600,1; 24*3600,1]
+    "Plant enabling schedule allowing operators to lock out the plant during off-hour";
+  parameter Modelica.SIunits.Temperature TChiLocOut=277.5
+    "Outdoor air lockout temperature below which the chiller plant should be disabled";
+  parameter Modelica.SIunits.Time plaThrTim = 15*60
```

Comment:
Wouldn't it be easier to use enable and disable delay, or minimum on and off time for this and the parameter below?